tardis/energy_input/GXPacket.py

```diff
@@ -32,6 +33,8 @@ class GXPacketStatus(IntEnum):
     ("shell", int64),
     ("time_current", float64),
     ("tau", float64),
+    ("Z", int64),
```

Comment:
I get the feeling that using the same infrastructure as the tracker is better than attaching source to each packet @andrewfullard

Reply:
I agree.